pytorch_module_path: "/home/demo/Workspace/assembly_hole_segmentation"
weight_path: "/home/demo/catkin_ws/src/assembly_part_recognition/pretrained_weights/hole/hole_rgb_unet.pt"
rgb: /zivid_camera/color/image_color
width: 960
height: 600
cluster_thresh: 1
gpu_id: "0"
debug: true

